README.md

```diff
@@ -46,7 +46,7 @@ Other Style Guides
   1. [Accessors](#accessors)
   1. [Events](#events)
   1. [jQuery](#jquery)
-  1. [ECMAScript 5 Compatibility](#ecmascript-5-compatibility)
+  1. [ECMAScript 5 Compatibility](#es5-compat--kangax)
```

Comment:
```suggestion
  1. [ECMAScript 5 Compatibility](#ecmascript-5-compatibility)
```

Reply:
I already changed anchor link to the previous `#es5-compat--kangax` as you suggested before.
So, no need to change to the `#ecmascript-5-compatibility` here